content/docs/ethernet-v0.5/fpga.md

```diff
@@ -0,0 +1,283 @@
+    [    0.480000] UDP hash table entries: 256 (order: 1, 8192 bytes)
+    [    0.480000] UDP-Lite hash table entries: 256 (order: 1, 8192 bytes)
+    [    0.490000] NET: Registered protocol family 1
+    [    0.510000] RPC: Registered named UNIX socket transport module.
+    [    0.510000] RPC: Registered udp transport module.
+    [    0.510000] RPC: Registered tcp transport module.
+    [    0.510000] RPC: Registered tcp NFSv4.1 backchannel transport module.
+    [    2.770000] Unpacking initramfs...
+    [    5.040000] hid_keyboard address 40020000, remapped to ffffffff78002000
+    [    5.040000] hid_display address 40028000, remapped to ffffffff78010000
+    [    5.390000] console [xuart_console0] enabled
+    [    5.400000] keyb_timer is started
+    [    5.440000] futex hash table entries: 256 (order: 0, 6144 bytes)
+    [    5.460000] workingset: timestamp_bits=61 max_order=15 bucket_order=0
+    [    6.000000] io scheduler noop registered (default)
+    [    9.100000] lowrisc-digilent-ethernet: Lowrisc ethernet platform (40012000-40013FFF) mapped to ffffffff78008000
+    [    9.130000] libphy: GPIO Bitbanged LowRISC: probed
+    [    9.140000] Probing lowrisc-0:01
+    [    9.150000] SMSC LAN8710/LAN8720 lowrisc-0:01: attached PHY driver [SMSC LAN8710/LAN8720] (mii_bus:phy_addr=lowrisc-0:01, irq=-1)
+    [    9.200000] lowrisc_digilent_ethernet lowrisc_digilent_ethernet: Lowrisc Ether100MHz registered
+    [    9.300000] Card inserted, mask changed to 4
+    [    9.330000] NET: Registered protocol family 17
+    [    9.340000] Key type dns_resolver registered
+    [    9.360000] mmc0: mmc_rescan_try_freq: trying to init card at 5000000 Hz
+    [    9.530000] Freeing unused kernel memory: 856K (ffffffff80000000 - ffffffff800d6000)
+    [    9.550000] This architecture does not have kernel memory protection.
+    Running the initial fake init
+    [   10.030000] Open device, request interrupt
+    Setting the clock ...
+    [   10.910000] mmc0: new SDHC card at address 0007
+    [   10.930000] blk_queue_max_hw_sectors: set to minimum 8
+    [   10.950000] mmcblk0: mmc0:0007 SD08G 7.42 GiB 
+    [   11.020000]  mmcblk0: p1 p2 p3
+    Mounting the nfs partition ...
+    Switch to nfs root
+    INIT: version 2.88 booting
+    bootlogd: cannot find console device 4:0 under /dev
+    [   46.690000] random: dd urandom read with 103 bits of entropy available
+    [   60.370000] random: nonblocking pool is initialized
+    hwclock: can't open '/dev/misc/rtc': No such file or directory
+    INIT: Entering runlevel: 2
+    Configuring network interfaces... ifup skipped for nfsroot interface eth0
+    run-parts: /etc/network/if-pre-up.d/nfsroot: exit status 1
+    Starting Dropbear SSH server: dropbear.
+    hwclock: can't open '/dev/misc/rtc': No such file or directory
+    Starting syslogd/klogd: done
+
+    Poky (Yocto Project Reference Distro) 2.0+snapshot-20171219 qemuriscv64 /dev/console
+
+    qemuriscv64 login: 
```

Comment:
Need to explain that you can log in with username 'root' and no password. Same for fpga2.md